# Name of predicate
string predicate

# List of argument bindings
string[] bindings

# Whether predicate is negated
bool negated

# Truth value between 0.0 and 1.0
float32 truth